adobe/spectrum-css
# fix(rating): address inconsistent hover behavior

## Description

Provides more granular control over the hover behavior of child stars within the rating component to prevent hovering in space adjacent to the component from highlighting all stars.

CSS-735

## How and where has this been tested?

Verified in local Storybook story for rating component.

### Validation steps

1. Fetch the branch and run it locally or open the Storybook URL for the PR.
2. Navigate to the rating component.
3. Verify that hovering over rating stars is now more controlled and that hovering between stars doesn't result or all/no stars being displayed as active.
4. Verify that changing the selected star works as expected with the hover behavior.

### Regression testing

Validate:

1. The documentation pages for at least two other components are still loading, including:


5. If components have been modified, VRTs have been run on this branch:


## Screenshots

![Kapture 2024-10-02 at 11 07 46](https://github.com/user-attachments/assets/459e2e5d-7a68-4918-afd5-caa447c4aff9)

## To-do list
